### PR TITLE
Update eventhub scaler docs with information for non public clouds.

### PR DESCRIPTION
**Pull request by v-shenoy**
Signed-off-by: Vighnesh Shenoy <vshenoy@microsoft.com>

Update eventhub scaler docs with information for non public clouds.
https://github.com/kedacore/keda/pull/2788

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #
https://github.com/kedacore/keda/issues/1915

### CODE REVIEW

**Comment by JorTurFer** [comment]
Hi @v-shenoy 
Thanks for this!! Could you rebase main? There is a commit that solves Netlify error

File: content/docs/2.7/scalers/azure-event-hub.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ triggers:
     consumerGroup: $Default
     unprocessedEventThreshold: '64'
     blobContainer: 'name_of_container'
+    # Optional (Default: AzurePublicCloud)
+    cloud: private
+    # Required when cloud = Private
+    endpointSuffix: servicebus.airgap.example
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/azure-event-hub.md:26]: Please, remove the end whitespaces

File: content/docs/2.7/scalers/azure-event-hub.md [hunk 2]
```diff
@@ -34,6 +40,9 @@ triggers:
     - `blobMetadata` - For all implementations that store checkpoint information on blob metadata such as current C#, Python, Java and JavaScript Event Hub SDKs.
     - `goSdk` - For all implementations using the [Golang SDK](https://github.com/Azure/azure-event-hubs-go)'s checkpointing, for example Dapr.
     - When no checkpoint strategy is specified, the Event Hub scaler will use backwards compatibility and able to scale older implementations of C#, Python or Java Event Hub SDKs. (see "Legacy checkpointing"). If this behaviour should be used, `blobContainer` is also required.
+- `cloud` - Name of the cloud environment that the Event Hub belongs to. Must be a known Azure cloud environment, or `Private` for Azure Stack Hub or Air Gapped clouds. (valid values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `AzureGermanCloud`, `Private`; default: `AzurePublicCloud`)
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/azure-event-hub.md:43]: Could you use the convention "name - Description. (Values: `x`, `y`, `z`, Default: `y`, Optional, Extra Info)"
And it'd be awesome if you could update also the old lines (35 and 38) using that pattern

File: content/docs/2.7/scalers/azure-event-hub.md [hunk 3]
```diff
@@ -34,6 +40,9 @@ triggers:
...
+- `endpointSuffix` - Service Bus endpoint suffix of the cloud environment the eventhub belongs to, e.g. `servicebus.cloudapi.de` for `AzureGermanCloud`. Required when `cloud` is set to `private`.
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/azure-event-hub.md:44]: Should it be Event Hub endpoint instead of Service Bus?

**Review by JorTurFer — COMMENTED** [review]
Thanks for the contribution! 
some nits inline

File: content/docs/2.7/scalers/azure-event-hub.md (hunk 1 above)
Comment (v-shenoy) [reply content/docs/2.7/scalers/azure-event-hub.md:26]: > Hi @v-shenoy Thanks for this!! Could you rebase main? There is a commit that solves Netlify error

Sure thing.

**Review by zroubalik — APPROVED** [review]

**Pull request merged**